hack/cherry_pick_pull.sh

```diff
@@ -28,12 +28,13 @@ cd "${KUBE_ROOT}"
 declare -r STARTINGBRANCH=$(git symbolic-ref --short HEAD)
 declare -r REBASEMAGIC="${KUBE_ROOT}/.git/rebase-apply"
 
-if [[ "$#" -ne 2 ]]; then
-  echo "${0} <pr-number> <remote branch>: cherry pick <pr> onto <remote branch> and leave instructions for proposing pull request"
+if [[ "$#" -lt 2 ]]; then
+  echo "${0} <pr-number>... <remote branch>: cherry pick one or more <pr> onto <remote branch> and leave instructions for proposing pull request"
```

Comment:
I guess it's new enough, internal enough, etc, that a "breaking" change in ordering is fine. :) (We also detect non-branchlike things pretty easily, so a misordering isn't fatal.)

I'll flip it. The array nonsense is painful, and shift is easy. AFK right now, will push later.